From 068a83773525943889fca01ffb480e78d1f62357 Mon Sep 17 00:00:00 2001
From: Patrice Chotard <patrice.chotard@st.com>
Date: Tue, 6 Oct 2020 16:21:23 +0200
Subject: [PATCH 0084/1141] am64: dts: st: add syscfg node for stm32mp251

System configuration controller is mainly used to manage system related
settings.

Signed-off-by: Patrice Chotard <patrice.chotard@st.com>
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Change-Id: I5045b3a8e0ce5f9848b7ce1c5e8d91f4a62e81d0
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/179770
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -142,6 +142,12 @@
 			feature-domains = <&rifsc 156>;
 		};
 
+		syscfg: syscon@44230000 {
+			compatible = "st,stm32mp25-syscfg", "syscon";
+			reg = <0x44230000 0x10000>;
+			clocks = <&scmi_clk CK_SCMI_SYSCPU1>;
+		};
+
 		/* Break node order to solve dependency probe issue between pinctrl and exti. */
 
 		pinctrl: pinctrl@44240000 {
